tests/elfs/static.ld

```diff
@@ -0,0 +1,28 @@
+PHDRS {
+    /*header PT_PHDR FILEHDR PHDRS;*/
+    prog PT_LOAD FLAGS(5);
+    stack PT_GNU_STACK FLAGS(6);
+    heap PT_NULL FLAGS(6);
+}
+
+SECTIONS {
+    .text 0x100000000 : {
+        *(.text*)
+    } :prog
+    .rodata : {
+        *(.rodata*)
+    } :prog
+    .stack 0x200000000 (NOLOAD) : {
+        . = .;
+    } :stack
+    .heap 0x300000000 (NOLOAD) : {
```

Comment:
Do we support heap sections?

Reply:
i think this file and static.elf should probably be removed now. I think they were used to drive initial development